src/main/resources/application.properties

```diff
@@ -23,9 +23,9 @@ openapi.version=@project.version@
 # Sort operations by method
 springdoc.swagger-ui.operationsSorter=method
 
-#===============================================================================
+#========================================
```

Comment:
The previous version (length) was better, since had the same lenght as all other dividers.